Source/DafnyCore/Rewriters/PrecedenceLinter.cs

```diff
@@ -107,27 +111,27 @@ class PrecedenceLinterVisitor : TopDownVisitor<LeftMargin> {
       // that is, we inspect line and column information.
 
       if (expr is BinaryExpr bin && (bin.Op == BinaryExpr.Opcode.Imp || bin.Op == BinaryExpr.Opcode.Exp || bin.Op == BinaryExpr.Opcode.Iff)) {
+        // For
```

Comment:
Was this suggested by `dotnet format` or something like that?

No objection to the comment move, but personally I find the nested ternary operator much more of a readability issue. :)

Reply:
No, I had to format it that way to bypass this bug: https://github.com/dotnet/format/issues/1566